.github/CODEOWNERS

```
@@ -0,0 +1 @@
+@clrcrl
```

I think [the syntax here](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) needs to be:
```suggestion
*       @clrcrl
```